### ISSUE TITLE
Focus indicator on summary details page (accordion)

### ISSUE THREAD

**Issue by InHuOfficial** [jfbrennan/m-#62]
https://m-docs.org/accordion

The accordion (summary and details) example on the page does not have visible focus indicators. 

They are being overridden in the theme with 

``` html
button:active, button:focus, details>summary:focus {
    outline: 0;
}
```

Additionally (sorry to mix reports) there is an error in the JS on the page.

``` 
Uncaught SyntaxError: Invalid or unexpected token. File: accordion: 171

<code-sample readonly="readonly">
  // Imperative
  <script>
    const handleToggle = e => \{ <---- this is leaking into the DOM
```

Chrome latest stable  92.0.4515.131

I understand that the example does not auto close other items and is just wrapped summary and details elements, so I don't think it has any impact of the page functionality but I thought I would let you know!

**Comment by jfbrennan** [comment]
adds focus-visible style to the summary elements in an accordion
